### ISSUE TITLE
The bot should try an alternative nick on numeric 437 

### ISSUE THREAD

**Issue by nyuszika7h** [ProgVal/Limnoria#1033]
```
INFO 2015-01-28T11:50:52 Creating new Irc for freenode.
INFO 2015-01-28T11:50:52 Connecting to rajaniemi.freenode.net:6697.
INFO 2015-01-28T11:50:52 Loading plugins (connecting to freenode).
ERROR 2015-01-28T11:51:01 Unhandled error message from server: IrcMsg(prefix="rajaniemi.freenode.net", command="437", args=('*', 'Caroline', 'Nick/channel is temporarily unavailable'))
INFO 2015-01-28T11:51:01 freenode: Server acknowledged capability 'account-notify'
INFO 2015-01-28T11:51:01 freenode: Server acknowledged capability 'extended-join'
INFO 2015-01-28T11:51:01 freenode: Server acknowledged capability 'multi-prefix'
INFO 2015-01-28T11:51:01 freenode: Server acknowledged capability 'sasl'
INFO 2015-01-28T11:51:02 freenode: Authenticating using SASL.
INFO 2015-01-28T11:51:04 freenode: SASL authentication successful
WARNING 2015-01-28T11:51:24 Error message from freenode: Closing Link: server.redray.es (Connection timed out)
```

**Comment by jamessan** [comment]
Based on my reading, the server shouldn't be sending 437 at that point.  433 seems more appropriate.

**Comment by nyuszika7h** [comment]
@jamessan The nick is not in use, it's being held by services with TS6 `NICKDELAY`, which happens when either:

* the account has `ENFORCE` set, and someone failed to identify to it in time
* the account's owner used the `RELEASE` command

**Comment by GLolol** [comment]
@jamessan's above commit was merged in 487f8c8af54bc2c92684f414e3591495b5852b57. Is this issue fixed?

**Comment by GLolol** [comment]
This was merged and fixed over 2 years ago
